MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -9,6 +9,7 @@ protocol FeedViewControllerDelegate {
 }
 
 public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching, FeedLoadingView {
+	@IBOutlet var errorView: ErrorView?
```

Comment:
This property setter can be private to prevent clients from changing the instance:

```suggestion
	@IBOutlet private(set) var errorView: ErrorView?
```